Firmware/ultralcd.cpp

```diff
@@ -1761,75 +1759,39 @@ void lcd_print_target_temps_first_line(){
     }
 }
 
-static void mFilamentPrompt()
+static void waitFilamentLoop()
 {
-uint8_t nLevel;
+    for(;;) {
+        manage_heater();
+        manage_inactivity(true);
```

Comment:
Note: lcd updates are enabled, so we probably don't want to call `lcd_update(0)`.

I'm not sure if we should disable lcd updates all together here.